appengine/pond_storage/opponents.py

```diff
@@ -194,7 +219,7 @@ def get_opponents(user_entry):
   duck_key = ndb.Key(urlsafe=urlsafe_key)
   duck = duck_key.get()
   user_entry = duck.leaderboard_entry_key.get()
-  opponents = get_opponents(user_entry)
+  opponents = get_opponents(user_entry, _OPPONENTS_NEEDED)
```

Comment:
This is a global so shouldn't need to passed in

Reply:
I did a refactor to pass in how many opponents to return to get_opponents. This was so that when we use get_opponents for ranking, we would have flexibility to request for a number different from the 3 needed for practice matches.

Reply:
I did a refactor to pass in how many opponents to return to get_opponents. This was so that when we use get_opponents for ranking, we would have flexibility to request for a number different from the 3 needed for practice matches.